client/scss/utilities/_variables.scss

```diff
@@ -203,6 +203,11 @@ $article-top-space: 2 * $vertical-space-unit;
 $border-radius-unit: 6px;
 $spacing-directions: ('top', 'right', 'bottom', 'left');
 
+$grid-padding-s: map-get(map-get($grid-config, 's'), 'padding');
```

Comment:
I made these as abstractions to make a clearer/more concise means of accessing the padding values at different breakpoints. Just feels less messy to write `$grid-padding-s` when we want the small grid padding, rather than `map-get(map-get($grid-config, 's'), 'padding')` (which requires a bit more cognitive overhead/increases probability of making/missing silly mistakes).